piety-promoted-v2/modernized/11-chapter-10.adoc

```diff
@@ -143,8 +143,12 @@ She died at the house of William Young, of Droinfield, in Derbyshire, of a consu
 in the year 1702, aged about thirty years,
 and was buried in Friends`' burying-ground there.
 
+[.embedded-content-document.testimony]
+--
+
+[.letter-heading]
```

Comment:
Let's switch this to be a secondary sub-heading, as discussed in slack.  So `=== ` to start, and combine the lines. 👍